src/Authenticator/JwtAuthenticator.php

```diff
@@ -32,7 +31,7 @@ class JwtAuthenticator extends TokenAuthenticator
         'queryParam' => 'token',
         'tokenPrefix' => 'bearer',
         'algorithms' => ['HS256'],
-        'entityClass' => Entity::class,
+        'entityClass' => '\Cake\ORM\Entity',
```

Comment:
It doesn't hurt either? So whats the standard for this if stickler doesn't complain?